Polygon_mesh_processing/doc/Polygon_mesh_processing/PackageDescription.txt

```diff
@@ -258,6 +258,7 @@ The page \ref bgl_namedparameters "Named Parameters" describes their usage.
 - `CGAL::Polygon_mesh_processing::triangle()`
 - `CGAL::Polygon_mesh_processing::region_growing_of_planes_on_faces()`
 - `CGAL::Polygon_mesh_processing::detect_corners_of_regions()`
+- `CGAL::Polygon_mesh_processing::refine_mesh_at_isolevel()`
```

Comment:
It would probably make sense to put it in corefinement instead of Miscellaneous.

Reply:
It reflects the license picked.